src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -320,9 +321,9 @@ private void SetupRecordingService()
                 recordingVisualPrefab = MobileRecordingSettings.Instance.OverrideMobileRecordingServicePrefab;
             }
 
-            if (MobileRecordingSettings.IsInitialized && 
-                MobileRecordingSettings.Instance.EnableMobileRecordingService &&
-                recordingVisualPrefab != null)
+            bool enableMobileRecordingService = (!MobileRecordingSettings.IsInitialized) || MobileRecordingSettings.Instance.EnableMobileRecordingService;

```

Comment:
Seems to me that if I don't have settings to explicitly disable recording, we should default to having recording (and the UX) on, right?  As it was formerly coded, if someone didn't add explicit settings, the mobile recording and status text wouldn't show up.

Reply:
This is a good point. We may want to clean up these functions better now that we have overloaded the mobile recording visual to contain items other than recording content.

---
In reply to: [343313429](https://github.com/microsoft/MixedReality-SpectatorView/pull/270#discussion_r343313429) [](ancestors = 343313429)